dhis-2/dhis-support/dhis-support-external/src/main/java/org/hisp/dhis/external/conf/ConfigurationKey.java

```diff
@@ -710,6 +713,9 @@ public enum ConfigurationKey {
   /** Enable TOTP-based 2FA authentication. (default: true) */
   TOTP_2FA_ENABLED("login.security.totp_2fa.enabled", Constants.ON, false),
 
+  /** Use read replica database for tracker queries. (default: false) */
+  TRACKER_READ_REPLICA_ENABLED("tracker.read_replica.enabled", Constants.OFF, false),
```

Comment:
What is the reason for two config keys? With two you end up having to deal with the case that its enabled but there is no url set. Users can toggle the usage of this feature by commenting the `tracker.read_replica.connection.url`. Or do you foresee some more configs related to this?

If we keep it group the configs. Why is this one down here.

Reply:
Good point. Yes initially there were more granular configuration related to this (mentioned in previous comment above) . But removed the others due to lack of time. But in general I still want to keep the main "enabling" config as the main toggler of the feature. I feel this makes it very explicit and the intentions are clear by a system admin. 

I'll group it together and I also foresee more configs coming for this (if we decide to proceed this route)